Proposals/xgov-26.md

```diff
@@ -0,0 +1,211 @@
+---
+title: ASA Stats Offer - Algorand blockchain ASA/NFT creators' channel
+author: Ivica Paleka (@ipaleka), Marcin Zawiejski (@dragmz), Eduard Ravnic (@kerrilija)
+company_name: ASA Stats
+category: Tools
+focus_area: User Onboarding
+open_source: Yes
+amount_requested: 250000
```

Comment:
If your project requires multiple milestones, split them into multiple proposals. See: https://github.com/algorandfoundation/ARCs/pull/209

Reply:
> If your project requires multiple milestones, split them into multiple proposals. See: [algorandfoundation/ARCs#209](https://github.com/algorandfoundation/ARCs/pull/209)

Afaic that's contrary to the guideline from the [provided template](https://github.com/algorandfoundation/ARCs/blob/main/assets/arc-0034/TemplateForm.md#roadmap):

<img src="https://github.com/algorandfoundation/xGov/assets/49662536/b1acbc8a-bbff-493e-b03d-9e32a4a0e6ff" width="720">

Reply:
So here are first three milestones from the proposal:

-----

### Milestone 1: Proof of concept for the user authentication

Date: 31/10/2023  
Amount: 2,000  
Description: Research existing technologies and possible ways for creating a transaction that will prove the user's ownership of an Algorand address. Create a working prototype.

### Milestone 2: Proof of concept for the atomic transfers to and from the escrows

Date: 31/12/2023  
Amount: 8,000  
Description: Research existing technologies and possible ways for creating the escrows and signing the atomic transfers from and to them from the user accounts. Define limits for the number of different ASA that can be involved in an offer and propose the optimal and maximum numbers. Create a working prototype.

### Milestone 3: Proof of concept for the non-users' notification transaction creation

Date: 31/10/2023  
Amount: 2,000  
Description: If the target address - to which the offer refers - hasn't been connected to an ASA Stats user then a transaction should be sent to the target address with the minimal amount of ALGO needed to trigger a notification on major wallet apps. Research, develop, and deploy a prototype.

-----

So should I rewrite proposal in a way that every milestone consists of work that lasts for about 3 (or multiplier) months?

Like:

-----
```

---
title: ASA Stats Offer - Algorand blockchain ASA/NFT creators' channel
author: Ivica Paleka (@ipaleka), Marcin Zawiejski (@dragmz), Eduard Ravnic (@kerrilija)
company_name: ASA Stats
category: Tools
focus_area: User Onboarding
open_source: Yes
amount_requested: 12000
status: Final
---
```
...

### Milestone 1: Proof of concepts

Date: 3 months  
Amount: 12,000  
Description: Research existing technologies and possible ways for:
 - define a process for creating a transaction that will prove the user's ownership of an Algorand address. Create a working prototype.
 - define a process for creating the escrows and signing the atomic transfers from and to them from the user accounts. Define limits for the number of different ASA that can be involved in an offer and propose the optimal and maximum numbers.
 - define a process for creating a transaction should be sent to the target address with the minimal amount of ALGO needed to trigger a notification on major wallet apps

### Milestone 2: ...
Date: 3 months  
Amount: 25,000  

### Milestone 3: ...
Date: 6 months  
Amount: 58,000  

### Milestone 4: ...

-----